docs/src/guide/inverting.md

```diff
@@ -6,12 +6,16 @@ existence of a forward index in the path `path/to/forward/cw09b`:
 
     $ mkdir -p path/to/inverted
     $ ./invert -i path/to/forward/cw09b \
-        -o path/to/inverted/cw09b \
-        --term-count `wc -w < path/to/forward/cw09b.terms`
+        -o path/to/inverted/cw09b
 
-Note that the script requires as parameter the number of terms to be
-indexed, which is obtained by embedding the
-`wc -w < path/to/forward/cw09b.terms` instruction.
+Note that the `invert` command requires the number of terms in the lexicon
+(or vocabulary) when building an inverted index. In the above example, the
+`invert` program assumes that a `cw09b.termlex' file exists from the
+output of `parse_collection` which is used to lookup the term count.
+
+The `invert` command also has a legacy option where the term count can be
```

Comment:
Instead of calling it a legacy option, I would simply mention that the number of terms can be provided using `--term-count` in case the lexicon is not available or on a different path.

In the future, we should try to standardize paths but for now, lots of our CLI tools don't make assumptions on what paths are. It's a bit of a mess unfortunately.

Not sure if you've seen it, but we wrote this [CLI wrapper](https://github.com/pisa-engine/pisa/tree/main/pisactl) for the last SIGIR tutorial, which is an POC of how this may look, but it's not necessarily the final solution (written in Python for now but ultimately we would like to improve the native binaries instead).